OpenSim/Common/Exception.h

```diff
@@ -67,14 +110,31 @@ class OSIMCOMMON_API Exception  : public std::exception {
     std::string _file;
     /** Line number at which the error occurred. */
     int _line;
+    /** Delimiter/separator used to separate different parts of the message.  */
+    static const std::string _msgSeparator;
```

Comment:
It's not *very* complicated, it is just unnecessary. I wouldn't care if it weren't in the Exception base class as a data member. How about just making it an anonymous namespace constant in the `.cpp` file rather than a static member? Then it is not part of the interface.

Possible lost in the discussion above: can the existing Exception data members be private now?